python/ray/autoscaler/gcp/node_provider.py

```diff
@@ -0,0 +1,194 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from googleapiclient import discovery
+
+from ray.autoscaler.node_provider import NodeProvider
+from ray.autoscaler.gcp.config import wait_for_compute_zone_operation
+from ray.autoscaler.gcp.tags import TAG_KEYS, TAG_VALUES
+from ray.ray_constants import BOTO_MAX_RETRIES
+
+TERMINATED_STATES = (
+    'STOPPING',
+    'STOPPED',
+    'SUSPENDING',
+    'SUSPENDED',
+    'TERMINATED',
```

Comment:
Is this the same state enum as here? https://cloud.google.com/compute/docs/instances/checking-instance-status

(I don't see suspending).

Reply:
The set of states was taken from the [instances.get](https://cloud.google.com/compute/docs/reference/rest/beta/instances/get) documentation. It says:
> [Output Only] The status of the instance. One of the following values: PROVISIONING, STAGING, RUNNING, STOPPING, STOPPED, SUSPENDING, SUSPENDED, and TERMINATED.